.github/workflows/ai-runner-docker.yaml

```diff
@@ -17,7 +17,7 @@ on:
 
 concurrency:
   group: ${{ github.workflow }}-${{ github.head_ref || github.run_id }}
-  cancel-in-progress: false
+  cancel-in-progress: ${{ !contains(github.ref, 'main') }}
```

Comment:
probably better to be precise, i'd suggest (including tags triggered runs):
```yaml
(github.ref_type == 'branch' && github.ref_name == 'main') || github.ref_type == 'tag'
```
and putting a not (`!`) clause on the whole condition.

Reply:
Good thinking! Was did what you had in mind https://github.com/livepeer/ai-worker/pull/338/commits/7abad1d43fb7d62840408ae82d35d75f5a43e11d?